group_vars/all.yml

```diff
@@ -164,7 +164,7 @@ proxy_setup: "{{ lookup('env','ORA_PROXY_SETUP') | lower | default('false',true)
 u01_lun: "{{ lookup('env','ORA_U01_LUN') | lower | default('',true) }}"
 
 #  Environment and infrastructure values (requirements) used for readiness checks
-minimum_ansible_version: 2.9
+minimum_ansible_version: "{{ lookup('file', playbook_dir + '/../.tool-versions') | regex_search('^ansible\\s+(.*)$', '\\1') | first }}"
```

Comment:
I think I've changed my mind here:  we don't change ansible minima often, and it looks like the file logic might not always work, so can we just go back to hard-coding to 2.9?  Thanks!